BungeeCord-Patches/0062-Add-command-to-view-all-proxy-plugins.patch

```diff
@@ -0,0 +1,79 @@
+From 481b64d2bac4530c108e0116666c91db2d6526a5 Mon Sep 17 00:00:00 2001
+From: MrFishCakes <fishcake007@outlook.com>
+Date: Fri, 21 May 2021 17:32:13 +0100
+Subject: [PATCH] Add command to view all proxy plugins
+
+
+diff --git a/proxy/src/main/java/io/github/waterfallmc/waterfall/command/CommandPlugins.java b/proxy/src/main/java/io/github/waterfallmc/waterfall/command/CommandPlugins.java
+new file mode 100644
+index 00000000..f97e37ab
+--- /dev/null
++++ b/proxy/src/main/java/io/github/waterfallmc/waterfall/command/CommandPlugins.java
+@@ -0,0 +1,44 @@
++package io.github.waterfallmc.waterfall.command;
++
++import net.md_5.bungee.BungeeCord;
++import net.md_5.bungee.api.ChatColor;
++import net.md_5.bungee.api.CommandSender;
++import net.md_5.bungee.api.chat.BaseComponent;
++import net.md_5.bungee.api.chat.ComponentBuilder;
++import net.md_5.bungee.api.plugin.Command;
++import net.md_5.bungee.api.plugin.Plugin;
++
++import java.util.Collection;
++
++public class CommandPlugins extends Command {
++
++    public CommandPlugins() {
++        super("gplugins", "bungeecord.command.plugins", "gpl");
++    }
++
++    @Override
++    public void execute(CommandSender sender, String[] args) {
++        sender.sendMessage(getPluginList());
++    }
++
++    private BaseComponent[] getPluginList() {
++        final Collection<Plugin> plugins = BungeeCord.getInstance().getPluginManager().getPlugins();
++        final ComponentBuilder builder = new ComponentBuilder();
++
++        builder.append("Plugins (" + plugins.size() + "): ");
++        boolean firstIteration = true;
++
++        for (Plugin plugin : plugins) {
++            if (firstIteration) {
++                firstIteration = false;
++            } else {
++                builder.append(ChatColor.WHITE.toString()).append(", ");
```

Comment:
You set the color not by prepending the ChatColor toString, but by calling .color(...) after append

Reply:
Now fixed.